posts/2020-05-15-five-years-of-rust.md

```diff
@@ -0,0 +1,508 @@
+---
+layout: post
+title: "Five Years of Rust"
+author: The Rust Core Team
+---
+
+With all that's going on in the world you'd be forgiven for forgetting that as
+of today, it has been five years since we released 1.0 in 2015! Rust has changed
+a lot these past five years, so we wanted reflect back on all of our
+contributors' work since the stabilization of the language.
+
+**If this is the first time you're reading about Rust:** Rust is a general
+purpose programming language empowering everyone to build reliable and efficient
+software. Rust can be built to run anywhere in the stack, whether as the kernel
+for your operating system or your next web app. It is built entirely by an open
+and diverse community of individuals, primarily volunteers who generously donate
+their time and expertise to help make Rust what it is.
+
+## Major Changes since 1.0
+
+#### 2015
+
+**[1.2] — Parallel Codegen** Compile time improvements are large theme to every
```

Comment:
As a matter of style/typesetting, I feel like there should be *something* between the paragraph title and the text after it. In my own writing, I use a period (`.`) for this purpose; sometimes `:` can also do.

But I can understand if others disagree, just wanted to bring it up.